cdap-watchdog/src/main/java/io/cdap/cdap/metrics/process/loader/MetricsWriterExtensionLoader.java

```diff
@@ -50,10 +56,22 @@
   @Inject
   public MetricsWriterExtensionLoader(CConfiguration cConf) {
     super(cConf.get(Constants.Metrics.METRICS_WRITER_EXTENSIONS_DIR));
+    String enabledExtensions = cConf.get(Constants.Metrics.METRICS_WRITER_EXTENSIONS_ENABLED_LIST);
+    if (enabledExtensions == null) {
+      this.enabledMetricsWriters = Collections.emptySet();
+      return;
+    }
+    String[] split = enabledExtensions.split(",");
```

Comment:
nit: consider adding an info log. I assume this happens once during startup and doesn't spam log

Reply:
or debug log

Reply:
Yes, this is called during startup and will not spam the logs. Added additional debug logs.